File: app/src/main/java/com/chooloo/www/callmanager/ui/activity/settings/SettingsContract.java
package com.chooloo.www.callmanager.ui.activity.settings;

import com.chooloo.www.callmanager.ui.base.BaseContract;

public class SettingsContract implements BaseContract {

    interface View extends BaseContract.View {
        void setUp();
    }

    interface Presenter<V extends View> extends BaseContract.Presenter<V> {

    }
}

File: app/src/main/java/com/chooloo/www/koler/ui/settings/SettingsContract.kt
package com.chooloo.www.koler.ui.settings

import android.view.MenuItem
import com.chooloo.www.chooloolib.ui.base.BaseContract

interface SettingsContract : BaseContract {
    interface View : BaseContract.View {
    }

    interface Controller<V : View> : BaseContract.Controller<V> {
        fun onMenuItemClick(menuItem: MenuItem)
    }
}